src/go.mod

```diff
@@ -3,15 +3,17 @@ module github.com/cloudfoundry/bosh-system-metrics-server
 go 1.20
 
 require (
-	github.com/golang/protobuf v1.2.0
+	github.com/golang/protobuf v1.5.3
```

Comment:
Probably not a big deal, but this protobuf library is deprecated.

Reply:
yup, go get -u warned me about that. Do I you think we need to complete that yak shave before bumping these dependencies?

Reply:
I started shaving the yak. 

Am I getting this right that `google.golang.org/grpc` still depends on `github.com/golang/protobuf` and that's OK (for reasons?) so it showing up as an indirect dependency is fine? The warning as gone away at least.

I got `protoc  --go_out=. --go_opt=paths=source_relative events.proto server.proto` to generate new .pb.go's, but those are not compatible with the current codebase:
```
pkg/egress/server.go:119:87: undefined: definitions.Egress_BoshMetricsServer
```

I'd like to stop shaving this yak since as you say we've deprecated the release